features/joinroom/impl/src/main/kotlin/io/element/android/features/joinroom/impl/JoinRoomNode.kt

```diff
@@ -43,14 +43,14 @@ class JoinRoomNode @AssistedInject constructor(
             state = state,
             onBackClick = ::navigateUp,
             onJoinSuccess = ::navigateUp,
-            onCancelKnockSuccess = ::navigateUp,
-            onKnockSuccess = { },
+            onCancelKnockSuccess = {},
+            onKnockSuccess = {},
```

Comment:
Maybe for later, but I believe that the parameters `onCancelKnockSuccess` and `onKnockSuccess` could be removed.